src/parameters/parameters_ode_solver.h

```diff
@@ -61,12 +62,15 @@ class ODESolverParam
         euler_ex, ///Forward Euler
         euler_im, ///Implicit Euler
         dirk_2_im, ///Second-order diagonally-implicit RK
-        dirk_3_im ///Third-order diagonally-implicit RK
+        dirk_3_im, ///Third-order diagonally-implicit RK
+        RK4_3_5_3SStar,
```

Comment:
Add short comments with the main properties - include order, number of stages, FSAL, and other pertinent information.